src/univariate/continuous/chi.jl

```diff
@@ -79,6 +79,13 @@ function mode(d::Chi; check_args::Bool=true)
     sqrt(ν - 1)
 end
 
+function kldivergence(p::Chi, q::Chi)
+    pν = first(params(p))
+    qν = first(params(q))
```

Comment:
Ah, I didn't realize it returns a tuple. Maybe in this case
```suggestion
    pν = dof(p)
    qν = dof(q)
```
would be a bit simpler and even more descriptive.